### ISSUE TITLE
--paper-input-container-prefix is documented but not implemented.

### ISSUE THREAD

**Issue by bendavis78** [PolymerElements/paper-input#180]
There is no `@apply` for `--paper-input-container-prefix`.

**Comment by bendavis78** [comment]
Actually, there appear to be three documented mixins that aren't implemented:

* --paper-input-container-input-disabled
* --paper-input-container-prefix
* --paper-input-container-suffix

In the stylesheet, they're implemented as:

* --paper-input-container-disabled
* --paper-input-prefix
* --paper-input-suffix

I'm not sure which of these are incorrect, the documentation or the code. My assumption is the latter, since all mixins in the docs use `--paper-input-container-*`.

**Comment by notwaldorf** [comment]
I think the implementation is correct, at least for the suffix and the prefix (there's a `paper-input-error` mixin for the error as well, which doesn't have the container). Finally, `--paper-input-container-disabled` in particular is applied to the whole container, not just the input, so i think the documentation has a typo in that case as well.

TLDR: code is right, docs are wrong. Want to fire a PR for that, or should i? :)

**Comment by bendavis78** [comment]
The convention I'm seeing is that all polymer elements define their own mixins using the element's name as a prefix. `--paper-input-error` is a mixin applied to the `:host` of the `paper-input-error` element (another convention is to use the element's name to mixin properties at that element's `:host` level). Since this is `paper-input-container`, and the prefix/suffix styles are applied within this element (and not in `paper-input`), I'm inclined to think the docs are correct.

**Comment by notwaldorf** [comment]
Unfortunately I think that changing the current mixin names might break existing code that is using it, so I'd like to not do that. So even though it's not 100% kosher, I'd rather have the documentation be slightly inconsistent for the prefix/suffix mixins, than break existing code.

**Comment by notwaldorf** [comment]
👉@cdata to double check he doesn't disagree

**Comment by jacobevans** [comment]
I second changing the docs.
Sincerely,
A user who just wasted 20 minutes trying to get this to work with the wrong mixin name :smile:
